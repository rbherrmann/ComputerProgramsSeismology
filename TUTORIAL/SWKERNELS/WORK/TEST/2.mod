20.0000 6.1000 3.45 2.7300 1.000E-02 1.000E-02 0.00 0.00 1.00 1.00
