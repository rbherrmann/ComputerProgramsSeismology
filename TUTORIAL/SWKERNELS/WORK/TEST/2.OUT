 betmn, betmx:   3.4500000       4.6999998    
 L ga 0   14.200000      0.0    0.0000000      0.10
 L gb 0   14.200000      0.0   6.51620794E-04  0.10
 L ga 0   14.200000      0.0    0.0000000      0.10
 L gb 0   14.200000      0.0   6.51620794E-04  0.10
 R ga 0   14.200000      0.0   1.03395731E-04  0.10
 R gb 0   14.200000      0.0   6.13210083E-04  0.10
 R ga 0   14.200000      0.0   1.03395731E-04  0.10
 R gb 0   14.200000      0.0   6.13210083E-04  0.10
Dispersion  fit (vel)         std err  :    0.1927 (km/s)
Dispersion  fit (vel)    mean residual :    0.6880 (km/s)
Dispersion  fit (vel)    avg |residual|:    0.6880 (km/s)
Percent of Signal Power Fit (Disp)     :  96.80975% for    4 SW Obs
      Mode        Period       Observed      Predicted Observed Sigma
 L ga 0   14.200000      0.0    0.0000000      0.10
 L gb 0   14.200000      0.0   6.51620794E-04  0.10
 L C    0     14.1999998      4.0000000      3.5886006      0.1000000
 L ga 0   14.200000      0.0    0.0000000      0.10
 L gb 0   14.200000      0.0   6.51620794E-04  0.10
 L U    0     14.1999998      4.0000000      3.3735213      0.1000000
 R ga 0   14.200000      0.0   1.03395731E-04  0.10
 R gb 0   14.200000      0.0   6.13210083E-04  0.10
 R C    0     14.1999998      4.0000000      3.2176099      0.1000000
 R ga 0   14.200000      0.0   1.03395731E-04  0.10
 R gb 0   14.200000      0.0   6.13210083E-04  0.10
 R U    0     14.1999998      4.0000000      3.0683899      0.1000000
      Mode        Period       Observed      Predicted Observed Sigma
 L ga 0   14.200000      0.0    0.0000000      0.10
 L gb 0   14.200000      0.0   6.51620794E-04  0.10
 L G    0     14.1999998 0.40000000E+01 0.65162079E-03 0.10000000E+00
 L ga 0   14.200000      0.0    0.0000000      0.10
 L gb 0   14.200000      0.0   6.51620794E-04  0.10
 R ga 0   14.200000      0.0   1.03395731E-04  0.10
 R gb 0   14.200000      0.0   6.13210083E-04  0.10
 R G    0     14.1999998 0.40000000E+01 0.71660586E-03 0.10000000E+00
 R ga 0   14.200000      0.0   1.03395731E-04  0.10
 R gb 0   14.200000      0.0   6.13210083E-04  0.10
